libzeth/tests/core/evaluator_from_lagrange_test.cpp

```diff
@@ -44,21 +44,21 @@ TEST(EvaluationFromLagrangeTest, ComputeLagrangeEvaluation)
         std::map<size_t, Fr> l_factors;
         l_factors[j] = Fr::one();
 
-        G1 L_j_g1 = eval_g1.evaluate_from_lagrange_factors(l_factors);
-        ASSERT_EQ(L_j_g1, lagrange.lagrange_g1[j])
+        G1 l_j_g1 = eval_g1.evaluate_from_lagrange_factors(l_factors);
```

Comment:
See if we want to switch to `aNy_CasE` styling for variable names, especially for scenarios where we want to stick to notations of a paper